finance/server/src/main/java/com/google/finapp/ServerMain.java

```diff
@@ -54,6 +56,10 @@ private static SpannerDaoInterface getSpannerDao(
     DatabaseClient client =
         spanner.getDatabaseClient(
             DatabaseId.of(spannerProjectId, spannerInstanceId, spannerDatabaseId));
+
+    if (spannerUsePg) {
```

Comment:
nit: maybe move this above where we have a branch for jdbc.

Reply:
The main reason for this placement is that the `SpannerDaoPGImpl` class leverages the same java driver as `SpannerDaoImpl`, rather than JDBC. So it takes different parameters (primarily the `client`).

Reply:
See relevant comment about merging the use_pg and use_jdbc and make them an enum. Thoughts?

Reply:
as mentioned in other comment, probably need's @c24t comment regarding samples workflow